### ISSUE TITLE
faulty update 20221118

### ISSUE THREAD

**Issue by FoscoV** [lineageos4microg/lineageos4microg.github.io#106]
I updated a Sony XA2 to lineage-19.1-20221118-microG-pioneer from the previous lineage-19.1-20221022-microG-pioneer.
I relied on the system updater function, which worked smoothly up to now.
Something went wrong so that after reboot the device got stuck at the boot animation.
From recovery interface I tried reinstalling the update with no change (this get me to figure that everything worked as expected within the update process).
Downgrade is not allowed so I didn't reversed to 20221022.
I installed the update from vanilla lineageos (lineage-19.1-20221117-nightly-pioneer-signed), the recovery interface issued about the timestamp but not the OTA so that it was forceable. The boot worked and the system is now up and running (obvious minor issues relate to the miss of both GApps and microG).

Mismatching keys break system updater. I'm looking forward for next microg-lineageos update to install it (by recovery) and get back on rails.

I've no clue of what broke 20221118-microG but it seems to me that's not due to lineageos but it's on the forked build.

Leaving this here, while not sure this issue can fit.

**Comment by HaykCS01** [comment]
Any news?

**Comment by FoscoV** [comment]
The system works on the update from lineageOS (no Gapps), no major issue rised for that 
I'm waiting for the next update from microg-lineageos to install it from recovery menu.

**Comment by FoscoV** [comment]
Long story short: the update fixed the issue

Long version:
When the next update came available I downloaded it.
From recovery I side-loaded the update on top to the vanilla lineageos (no microg nor other Gapps-like) I had.
After reboot I had to fix a couple of minor leaks (e.g. magisk install, some user settings) but the main environment (apps, data, sd card) works fine.

NB:
While on the "transition" lineageos I tried to interfere as few as possible with system files: no new apps, no system update,...
Beside the sd card that was only restored moving back to microg-lineageos, most of apps and functionality keep working the whole time
